From 042ce85b485c91ce980dc165ffeeab665b6a2bbc Mon Sep 17 00:00:00 2001
From: "Tan, Raymond" <raymond.tan@intel.com>
Date: Tue, 10 Mar 2015 13:07:16 +0800
Subject: [PATCH 111/389] platform: x86: add intel-quark directory

This patch is to add intel-quark directory to under
drivers/platform/x86 directory tree to host the
platform driver and board data for various Quark platforms

Signed-off-by: Tan, Raymond <raymond.tan@intel.com>
---
 drivers/platform/x86/Kconfig              | 4 ++++
 drivers/platform/x86/Makefile             | 1 +
 drivers/platform/x86/intel-quark/Kconfig  | 0
 drivers/platform/x86/intel-quark/Makefile | 0
 4 files changed, 5 insertions(+)
 create mode 100644 drivers/platform/x86/intel-quark/Kconfig
 create mode 100644 drivers/platform/x86/intel-quark/Makefile

diff --git a/drivers/platform/x86/Kconfig b/drivers/platform/x86/Kconfig
index 71dbfb5..f94d703 100644
--- a/drivers/platform/x86/Kconfig
+++ b/drivers/platform/x86/Kconfig
@@ -721,6 +721,10 @@ config INTEL_IMR
 
 	  If you are running on a Galileo/Quark say Y here.
 
+if  X86_INTEL_QUARK
+source "drivers/platform/x86/intel-quark/Kconfig
+endif
+
 config IBM_RTL
 	tristate "Device driver to enable PRTL support"
 	depends on X86 && PCI
diff --git a/drivers/platform/x86/Makefile b/drivers/platform/x86/Makefile
index 9b87cfc..17a7699 100644
--- a/drivers/platform/x86/Makefile
+++ b/drivers/platform/x86/Makefile
@@ -53,6 +53,7 @@ obj-$(CONFIG_SAMSUNG_Q10)	+= samsung-q10.o
 obj-$(CONFIG_APPLE_GMUX)	+= apple-gmux.o
 obj-$(CONFIG_INTEL_RST)		+= intel-rst.o
 obj-$(CONFIG_INTEL_SMARTCONNECT)	+= intel-smartconnect.o
+obj-$(CONFIG_X86_INTEL_QUARK)	+= intel-quark/
 
 obj-$(CONFIG_PVPANIC)           += pvpanic.o
 obj-$(CONFIG_INTEL_BAYTRAIL_MBI)	+= intel_baytrail.o
diff --git a/drivers/platform/x86/intel-quark/Kconfig b/drivers/platform/x86/intel-quark/Kconfig
new file mode 100644
index 0000000..e69de29
diff --git a/drivers/platform/x86/intel-quark/Makefile b/drivers/platform/x86/intel-quark/Makefile
new file mode 100644
index 0000000..e69de29
-- 
1.9.3

